rke/k8s_version_info.go

```diff
@@ -199,9 +199,9 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 		},
 		// The Calico/Canal template in this version use functions that are only available in RKE v1.0.0 and up
 		// This version includes nodelocal dns only available in RKE v1.0.7 and up
-		"v1.18.3-rancher2-3": {
-			MinRancherVersion: "2.4.6-rc0",
-			MinRKEVersion:     "1.1.4-rc0",
+		"v1.18.6-rancher1-1": {
+			MinRancherVersion: "2.4.5-rc0",
+			MinRKEVersion:     "1.1.3-rc0",
```

Comment:
minRKEversion should be 1.1.4-rc0?

Reply:
For Rancher 2.4.5, we use RKE 1.1.3 because https://github.com/rancher/rancher/blob/v2.4.5/go.mod#L86